zone/gm_commands/show/currencies.cpp

```diff
@@ -56,6 +56,16 @@ void ShowCurrencies(Client *c, const Seperator *sep)
 		has_currency = true;
 	}
 
+	const int aa_points = t->GetAAPoints();
```

Comment:
Personally, I don't see AAs as a currency. Just my two cents...

Reply:
I can remove that, we just didn't have a way to view them as of right now.

Reply:
Good point. I recommend breaking it out but let's add that ability somewhere.

Reply:
I'll make a separate command for it.